### PR TITLE
vstart.sh: set logging for client and MDS when -d is passed

### PR DESCRIPTION
**Pull request by rishabh-d-dave**
Set debug log messages for client and MDS to 20 when option "--debug" is
passed to vstart.sh

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [x] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by dparmar18** [comment]
there's a backtrace in make check log that starts with:

```
1: (ceph::__ceph_assert_fail(char const*, char const*, int, char const*)+0x1b4) [0x7fa01b69a1b4]
```
 looks like a race. wondering if it's related to the change

File: src/vstart.sh [hunk 1]
```diff
@@ -1347,6 +1351,10 @@ else
         debug auth = 20
         debug mgrc = 20
         debug ms = 1'
+    CCLIENTDEBUG='
+        debug client = 20'
+    CMDSDEBUG='
+        debug mds = 20'
```
Comment (gregsfortytwo) [inline src/vstart.sh:1357]: Why not just stick this in the config like the others immediately above?

**Comment by rishabh-d-dave** [comment]
https://jenkins.ceph.com/job/ceph-pull-requests/110492/

**Comment by rishabh-d-dave** [comment]
jenkins test make check

**Review by vshankar — APPROVED** [review]

**Comment by vshankar** [comment]
> there's a backtrace in make check log that starts with:
> 
> ```
> 1: (ceph::__ceph_assert_fail(char const*, char const*, int, char const*)+0x1b4) [0x7fa01b69a1b4]
> ```
> 
> looks like a race. wondering if it's related to the change

That backtrace is from librbd

```
 1: (ceph::__ceph_assert_fail(char const*, char const*, int, char const*)+0x1b4) [0x7fa01b69a1b4]
 2: /home/jenkins-build/build/workspace/ceph-pull-requests/build/lib/libceph-common.so.2(+0x1420fff) [0x7fa01b699fff]
 3: (AsyncOpTracker::~AsyncOpTracker()+0x46) [0x7fa01b58f4a6]
 4: (librbd::ImageWatcher<librbd::ImageCtx>::~ImageWatcher()+0x69) [0x55956fc9b4c9]
 5: (librbd::ImageWatcher<librbd::ImageCtx>::~ImageWatcher()+0x19) [0x55956fc9b579]
 6: (librbd::ImageCtx::shutdown()+0x33) [0x55956fc636c3]
 7: (librbd::image::CloseRequest<librbd::ImageCtx>::finish()+0x1c) [0x55956fe5df0c]
 8: (librbd::image::CloseRequest<librbd::ImageCtx>::handle_flush_image_watcher(int)+0x35f) [0x55956fe5e2ff]
 9: (librbd::util::detail::C_CallbackAdapter<librbd::image::CloseRequest<librbd::ImageCtx>, &librbd::image::CloseRequest<librbd::ImageCtx>::handle_flush_image_watcher>::finish(int)+0x66) [0x55956fe5ee46]
```

This is vstart.sh change, so its not related.

**Comment by idryomov** [comment]
> That backtrace is from librbd

Most likely https://tracker.ceph.com/issues/53959.

**Pull request merged**